### ISSUE TITLE
Pretrained enhanced label set

### ISSUE THREAD

**Issue by bpottersta** [YuanGongND/psla#12]
As per README.md, the link under "(Optional) Step 2. Enhance the label of the balanced AudioSet training set"
this link doesn't exist:
``` [pretrained enhanced label set](https://github.com/YuanGongND/psla/blob/main/here) ```

Can anyone supply this file?

**Comment by YuanGongND** [comment]
This is a mistake in the Readme file. 

The enhanced label sets are at https://github.com/YuanGongND/psla#pretrained-enhanced-label-set, click the name of the file to download. 

-Yuan
